File: packages/@simpli/cli-scaffold/generator/template/src/model/response/EnderecoResp.ts
/**
 * EnderecoResp
 * @author martinlabs CRUD generator
 */
import {ID, Resource} from '@/simpli'
import {ResponseSerialize} from '@/simpli'

import Endereco from '@/model/Endereco'

export default class EnderecoResp extends Resource {
  readonly $endpoint: string = '/Crud/Endereco{/id}'

  @ResponseSerialize(Endereco)
  endereco: Endereco = new Endereco()

  get $id() {
    return this.endereco.$id
  }

  set $id(val: ID) {
    this.endereco.$id = val
  }
}

File: packages/@simpli/cli-server/generator/template/src/main/java/br/com/martinlabs/usecase/crud/response/EnderecoResp.kt
package br.com.martinlabs.usecase.crud.response

import io.swagger.annotations.ApiModel
import io.swagger.annotations.ApiModelProperty
import br.com.martinlabs.usecase.model.Endereco

@ApiModel(value = "EnderecoResp")
class EnderecoResp {

    var endereco: Endereco? = null
}
